### PR TITLE
[WT-190] feature/share-links-options-tests

### PR DESCRIPTION
**Pull request by CandelR**
- Added test related with shared link options in drive view.
   - Copy link
   - Open share settings 
   - Delete link 
- Fixed tests:
   - Change password test

### CODE REVIEW

File: src/use_cases/trash/recover-items-from-trash.ts [hunk 1]
```diff
@@ -100,19 +103,23 @@ async function afterMoving(itemsToRecover, destinationId) {
...
     });
   }
+
+  if (failedItems.length > 0) {
+    store.dispatch(storageActions.popItems({ updateRecents: true, items: failedItems }));
+    store.dispatch(storageActions.clearSelectedItems());
+  }
 }
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```
Comment (JoanVicens) [inline src/use_cases/trash/recover-items-from-trash.ts]: Less lint exceptions are always nice 🙌🏼

**Review by JoanVicens — APPROVED** [review]
Looks good to me

**Review by JoanVicens — APPROVED** [review]

File: src/use_cases/trash/move-items-to-trash.ts [hunk 2]
```diff
@@ -15,28 +14,26 @@ const moveItemsToTrash = async (itemsToTrash: DriveItemData[]): Promise<void> =>
...
   });
 
-  const trashClient = await SdkFactory.getNewApiInstance().createTrashClient();
-  await trashClient.addItemsToTrash({ items } as AddItemsToTrashPayload);
-
   store.dispatch(storageActions.popItems({ updateRecents: true, items: itemsToTrash }));
   store.dispatch(storageActions.clearSelectedItems());
 
+  const trashClient = await SdkFactory.getNewApiInstance().createTrashClient();
+  await trashClient.addItemsToTrash({ items } as AddItemsToTrashPayload);
```
Comment (fabioespinosa) [inline src/use_cases/trash/move-items-to-trash.ts]: Curious why are we modifying move-items-to-trash in the share links options here 🤔

File: test/e2e/cypress/integration/share-links.spec.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  it('Should copy the share link of a shared file', () => {
+    cy.get(DATA_TEST_FILE_LIST_FILE).eq(0).find(ID_DROPDOWN).click();
+    cy.contains('Copy link').click();
+
+    cy.window()
+      .its('navigator.clipboard')
+      .invoke('readText')
+      .then((text) => {
+        expect(text).contains(SHARED_FILE_URL_FRAGMENT);
```
Comment (fabioespinosa) [inline test/e2e/cypress/integration/share-links.spec.ts:29]: Didn't know you can check the clipboard, nice find

**Review by fabioespinosa — APPROVED** [review]
Nice job!

**Pull request merged**